app/ViewModels/Concerns/Transaction/InteractsWithVendorField.php

```diff
@@ -25,4 +27,19 @@ public function vendorField(): ?string
 
         return $vendorField;
     }
+
+    public function migratedAddress(): ?string
```

Comment:
Just been looking at the migrated address with Mici and Sam. could you make this `getMigratedAddressAttribute`, then calling with `$model->migratedAddress` or `$transaction->migratedAddress` should only run it once. Not necessarily related to the issue but saves calculating it multiple times :thinking:

Reply:
I added memoization for this in https://github.com/ArdentHQ/arkscan/pull/129/commits/77efbbe905bfe503fb6843a5b42954374958323c but since most of these "getter" methods are within the Transaction's View Model (the class that's using this trait), I'd also keep that method in here, to keep the Transaction model is pretty light like it is now.